app/javascript/styles/mastodon/components.scss

```
@@ -1170,7 +1170,7 @@ body > [data-popper-placement] {
     width: 0;
     position: absolute;
     top: 0;
-    inset-inline-start: 16px + ((46px - 2px) / 2);
+    inset-inline-start: calc(16px + ((46px - 2px) / 2));
```

I think it may be pointing to use https://sass-lang.com/documentation/modules/math#div instead of the `calc()` which would probably fall back to the CSS `calc*()` at runtime. Although I still maintain node-sass, my dart-sass modern syntax knowledge is poor

---

doing the actual math though, wouldn't this just be the constant value
```suggestion
    inset-inline-start: -6px;
```

---

https://sass-lang.com/documentation/values/calculations
> Sass will simplify these as much as possible, even if they're combined with one another.

This include simplifying to `38px`

> doing the actual math though, wouldn't this just be the constant value

yes, but those numbers don't come from nowhere, and it's more maintainable to have the logic behind